src/components/MainMenu.tsx

```diff
@@ -60,11 +67,13 @@ class MainMenu extends Component<Props, State> {
                   position: 'absolute',
                   top: '50px',
                   right: 0,
-                  zIndex: suomifiTheme.zindexes.focus + 1,
+                  zIndex: 10000,
```

Comment:
zIndexes were no longer available from the tokens and therefore a hardcoded value is now used instead.